react/Navbar.tsx

```diff
@@ -58,4 +77,15 @@ const Navbar: FunctionComponent<Props & InjectedIntlProps> = ({
   )
 }
 
+function splitLocale(locale: string) {
```

Comment:
Could just be:
```suggestion
const splitLocale = (locale: string) =>  locale.split('-')[0]
```
The code has the correct behavior, though. Changing or not is completely up to you 🙂

Reply:
Yep, good catch! 

I was using arrow functions right when I wrote this hahahah
I changed it to `function` just for better readability, as `function` allows me to put these declarations wherever I want in the file, which allowed me to declare these functions _after_ declaring the actual `FunctionComponent`, not before, as I would have to do if these were arrow functions.

Reply:
Oh, okay